frontend/src/actions/recycleBin.js

```diff
@@ -57,7 +57,6 @@ export const getDiscarded = (projectID) => {
                     const row = {
                         id: response.data[i].ID,
                         metadata: response.data[i].metadata,
-                        similarityPair: response.data[i].similarityPair,
```

Comment:
So when I put things in the recyclebin it looks like it's still calling for comparisons and then throwing an error:

in the backend logs
```
backend              | [02/Feb/2022 22:43:33] "GET /api/comparisons/undefined?text=Agricultural%20Mechanization,%20Other HTTP/1.1" 404 24899
```
In the frontend logs
```
SuggestedLabels.jsx:14 GET http://localhost:8000/api/comparisons/undefined?text=Agricultural%20Mechanization,%20Other 404 (Not Found)
(anonymous) @ SuggestedLabels.jsx:14
commitHookEffectListMount @ react-dom.development.js:19731
commitPassiveHookEffects @ react-dom.development.js:19769
callCallback @ react-dom.development.js:188
invokeGuardedCallbackDev @ react-dom.development.js:237
invokeGuardedCallback @ react-dom.development.js:292
flushPassiveEffectsImpl @ react-dom.development.js:22853
unstable_runWithPriority @ scheduler.development.js:653
runWithPriority$1 @ react-dom.development.js:11039
flushPassiveEffects @ react-dom.development.js:22820
(anonymous) @ react-dom.development.js:22699
workLoop @ scheduler.development.js:597
flushWork @ scheduler.development.js:552
performWorkUntilDeadline @ scheduler.development.js:164
SuggestedLabels.jsx:17 Error: Not Found
    at handleErrors (SuggestedLabels.jsx:5:15)
```